iaso/api/completeness_stats.py

```diff
@@ -136,6 +136,10 @@ def __init__(self, *args, **kwargs):
                 iaso_profile__account=user.iaso_profile.account
             ).distinct()
 
+            self.fields["project_ids"].child_relation.queryset = Project.objects.filter(
+                account=user.iaso_profile.account
+            ).distinct()
```

Comment:
There is a trap with `distinct()` ([quoted from Django doc](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#django.db.models.query.QuerySet.distinct)):

> For a normal `distinct()` call, the database compares each field in each row when determining which rows are distinct.
> For a `distinct()` call with specified field names, the database will only compare the specified field names.

So, if possible, I would use something like `distinct("id")` to avoid a potential performance issue.